packages/phrases-ui/src/locales/ru.ts

```diff
@@ -67,9 +67,9 @@ const translation = {
     social_link_phone: 'Вы можете привязать другой номер телефона',
     social_link_email_or_phone: 'Вы можете привязать другой адрес электронной почты или телефон',
     social_bind_with_existing: 'Мы нашли связанную учетную запись, вы можете связать ее напрямую.',
-    reset_password: 'Сбросить пароль',
```

Comment:
I asked ChatGPT which one is more correct, and this is the answer. lol.
Not an expert in Russian, so I'll let you decide. From what it said though, `Сбросить пароль` seems to be the right one, since we are going to `Reset` the password, not `Recover` it.

<img width="716" alt="image" src="https://user-images.githubusercontent.com/12833674/220228544-5bf6edf9-bdc9-40b2-b079-5a9cd81589b1.png">

Reply:
The phrase "Сбросить пароль" looked ugly, standing out. But you're right, "Восстановление пароля" is a completely different operation. Changed it to "Сброс пароля".

Also changed description "код для сброса" which means "code to reset" to "код для изменения" which means "code to change". Because in Russian, a lot of words with the same root in one place looks repulsive.